js/src/components/paid-ads/ads-campaign.js

```diff
@@ -60,24 +77,27 @@ export default function AdsCampaign( { campaign, onContinue } ) {
 					{
 						link: (
 							<AppDocumentationLink
-								context={
-									isCreation ? 'create-ads' : 'edit-ads'
-								}
+								context={ trackingContext }
 								linkId="see-what-ads-look-like"
 								href="https://support.google.com/google-ads/answer/6275294"
 							/>
 						),
 					}
 				) }
 			/>
-			{ isCreation ? (
-				<CreateCampaignFormContent formProps={ formContext } />
-			) : (
-				<EditPaidAdsCampaignFormContent
-					formProps={ formContext }
-					allowMultiple={ campaign.allowMultiple }
-				/>
-			) }
+			<AudienceSection
+				disabled={ ! isCreation }
+				multiple={ isCreation || campaign.allowMultiple }
```

Comment:
💅  Since `! isCreation` disables the component. I guess multiple will be always true when the component is enabled.

Reply:
When `allowMultiple` is false, it also renders another selector which matches the wanted visual result.